.gitignore

```diff
@@ -121,12 +121,9 @@ Tools/msi/obj
 Tools/ssl/amd64
 Tools/ssl/win32
 
-# TODO: Once we auto-regen frozem modules for Windows builds
-# we can drop the .h files from the repo and ignore them here.
-# At that point we will rely the frozen manifest file to identify
-# changed generated files.  We'll drop the entry for it then.
-# See: Tools/scripts/freeze_modules.py.
-#Python/frozen_modules/*.h
+# Python/frozen_modules/MANIFEST is enough to tell if the generated
+# files have changed.  See Tools/scripts/freeze_modules.py.
```

Comment:
Good point.  I don't mind dropping MANIFEST from the repo too.

Reply:
Would you still generate it? Why exactly? Just to satisfy that GitHub action? I'd kill the action too (or remove this part from it, if it's checking for other things as well).

Reply:
The GitHub action doesn't care about files that are not in the repo (and ignored).  I want to keep generating the manifest because I find it helpful to have a flat, clean list of all the frozen modules.  On several occasions that has been useful.  However, it isn't a huge benefit.  If you think it still isn't worth it then I'm okay with removing the generation code too.